CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### Features
 
+* `[jest-matcher-utils] Change "suggest to equal" message to be more advisory`
```

Comment:
@ggarek nit - the backtick here is in the wrong place

Reply:
Thank you! Fixed 👍